packages/core/content-releases/server/src/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+export const getService = (
+  name: 'release' | 'release-validation',
+  { strapi } = { strapi: global.strapi }
+) => {
+  return strapi.plugin('content-releases').service(name);
+};
```

Comment:
Do we love type safety via strapi.plugin, or is it smart enough?

Reply:
Yeah we lose type safety when getting services through the plugin API